README.md

```diff
@@ -554,24 +685,59 @@ curl -X POST \
 The response shows that both **Motion Sensor** data and **Smart Lamp** data are
 being persisted in the database.
 
-### List the first N Sampled Values
+### :four: List the first N Sampled Values
```

Comment:
Is this a duplicate of  one of the "Check we are persisting" methods? If so there is no need to repeat - delete the **First** example and add explanation to the text  "Note that this query could also be used to check that data is persisting to .... etc."

The reason is to drip feed the knowledge to the student rather than surprise them with too much new stuff at once.

Reply:
Not really the same one, but with the refactoring of the subscriptions you requested this is gone now.